submissions/Harnytskyi/html_css_popup/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="css/style.css">  
+        <link rel="icon" href="img/favicon.png">
+        <title>Popup</title>
+    </head>
+    <body>
+        <nav>
+            <ul class="menu">
+                <li class="menu-item">
+                    <a href="#">Mail</a>
+                </li>
+                <li class="menu-item">
+                    <a href="#">Images</a>
+                </li>
+                <li class="menu-item">
+                    <input class="popup-check" id="popup-check" type="checkbox" />
```

Comment:
Single tags should be without slashes "/" (according to HTML5 specification). https://codebra.ru/en/lessons-html/dating/2/1
Could you please remove it?